src/Symfony/Component/Notifier/Bridge/Mercure/MercureTransport.php

```diff
@@ -77,6 +77,8 @@ protected function doSend(MessageInterface $message): SentMessage
             '@context' => 'https://www.w3.org/ns/activitystreams',
             'type' => 'Announce',
             'summary' => $message->getSubject(),
+            'mediaType' => 'application/json',
+            'content' => $options->getContent(),
```

Comment:
To make it fit correctly it should be serialized to json here. (Only the values of the content entry)

For now I think it is enough to just inject the Serializer and always encode to json. To keep this PR simple.